x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_details_view.tsx

```diff
@@ -28,11 +28,11 @@ const EnrichmentSectionHeader: React.FC<{ type?: ENRICHMENT_TYPES }> = ({ type }
       <EuiFlexGroup direction="row" gutterSize="xs" alignItems="baseline">
         <EuiFlexItem grow={false}>
           <EuiTitle size="xxxs">
-            <h5>
+            <h3>
```

Comment:
shouldn't this be `h6`? Looking at the Title section on [EUI](https://eui.elastic.co/#/display/title), they're using `h6` with `<EuiTitle size="xxxs">`